rtd-requirements.txt

```diff
@@ -10,6 +10,8 @@
 
 Sphinx>=1.7.6,<2.0.0; python_version <= '3.4'
 Sphinx>=3.0.4,<4.0.0; python_version >= '3.5'
```

Comment:
we should replace the pinning to <4.0.0 to !=4.0.0

Reply:
Update: I now understand why the test run with minimum package levels on py39 does not fail: While it picks up Sphinx 3.0.4 which does not yet have the docutils fix, it also picks up the minimum version of docutils which is 0.13. In a real environment however, the combination of Sphinx 3.x and docutils >=0.18 could be encountered and then it would fail. So I suggest requiring Sphinx >=4.0.1 and probably also docutils >=0.18.

Reply:
I think that the issue is more than even that.  I do not know why Sphinx builds documents with any released version of Sphinx since the latest info on their site is that they will fix Sphinx to be compatible with docutils for Sphinx 4.3.0.  I suggest that we make docutils< 0.18 mandatory for all Sphinx versions until this settles down.